fp-proj/src/app/Components/contact/contact.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Testow nie bedziemy pisac (chyba ze zamierzasz pozniej dodac ale tego nie ma w programie poki co). wiec mozesz w angular.json w ustawieniach schematics wyłączyc generowanie testow dla komponentow, dyrektyw czy serwisow (jak to zrobic znajdziesz w dokumentacji :) - ale wystarczy ze ustawisz dla nich odpowiednia flage skipTests: true